pantable/pantable.py

```diff
@@ -177,14 +182,13 @@ def read_data(include, data):
     Return None when the include path is invalid.
     """
     if include is None:
-        if py2:
-            data = data.encode('utf-8')
```

Comment:
Would this line means that you don't need to change the tests to start with `u'...'`?